arch/lkl/defconfig

```diff
@@ -1,14 +1,11 @@
 # CONFIG_LOCALVERSION_AUTO is not set
-# CONFIG_USELIB is not set
 CONFIG_NO_HZ_IDLE=y
 # CONFIG_SYSFS_SYSCALL is not set
 CONFIG_KALLSYMS_ALL=y
 # CONFIG_BASE_FULL is not set
 # CONFIG_FUTEX is not set
-CONFIG_EPOLL=y
```

Comment:
Tthey are default in newer kernels, so there is no need to explicitly mentioned them in the saved defconfig:

$ find -name Kconfig | xargs grep -A2 EPOLL
./init/Kconfig:config EPOLL
./init/Kconfig-	bool "Enable eventpoll support" if EXPERT
./init/Kconfig-	default y